doc/quickstart/configure.rst

```diff
@@ -165,6 +174,155 @@ the user.
    ``--argument_name argument_value``
 
 
+.. _config-esgf:
+
+ESGF configuration
+==================
+
+The ``esmvaltool run recipe_example.yml`` command will automatically try
+to download the files required to run the recipe from ESGF for the projects
+CMIP3, CMIP5, CMIP6, CORDEX, and obs4MIPs.
+For downloading some files (e.g. those produced by the CORDEX project),
+you need to log in to be able to download the data.
+
+See the
+`ESGF user guide <https://esgf.github.io/esgf-user-support/user_guide.html>`_
+for instructions on how to create an ESGF OpenID account if you do not have
+one yet.
+Note that the OpenID account consists of 3 components instead of the usual
+two, in addition a username and password you also need the hostname of the
+provider of the ID; for example `esgf-data.dkrz.de`.
+
+Next, configure your system so esmvaltool can use your
+credentials. This can be done using the keyring_ package or
+they can be stored in a :ref:`configuration file <config_esgf_pyclient>`.
+
+.. _keyring:
+
+Storing credentials in keyring
+------------------------------
+First install the keyring package. Note that this requires a supported
+backend that may not be available on compute clusters, see the
+`keyring documentation <https://pypi.org/project/keyring>`__ for more
+information.
+
+.. code-block:: bash
+
+    pip install keyring
+
+Next, set your username and password by running the commands:
+
+.. code-block:: bash
+
+    keyring set ESGF hostname
```

Comment:
I find this section a bit confusing, mostly because ESGF has a bunch of hosts and how they manage the IDs. There are a few things that are not clear at first glance 

- Should I add only one or all I want to use?
- If only one, it can be anyone or must be the one I used to get my ESGF openID?
- The username is just the username or the full ESGF openID?

An example will be very helpful.

Reply:
I added an example and more explanation